systems/robotInterfaces/@Biped/planZMPTraj.m

```diff
@@ -45,6 +46,9 @@
 
 [zmp0, supp0] = getZMPBetweenFeet(biped, struct('right', steps.right(1), 'left', steps.left(1)));
```

Comment:
replace `zmp0` with `~` since it will be overwritten immediately on the next line